pkg/stanza/operator/input/windows/input.go

```diff
@@ -226,6 +203,14 @@ func (i *Input) read(ctx context.Context) int {
 			}
 			i.Logger().Info("Resubscribing, creating remote subscription")
 			i.subscription = NewRemoteSubscription(i.remote.Server)
+			if err := i.startRemoteSession(); err != nil {
+				i.Logger().Error("Failed to re-establish remote session", zap.String("server", i.remote.Server), zap.Error(err))
+				return 0
+			}
+			if err := i.subscription.Open(i.startAt, uintptr(i.remoteSessionHandle), i.channel, i.bookmark); err != nil {
+				i.Logger().Error("Failed to re-open subscription for remote server", zap.String("server", i.remote.Server), zap.Error(err))
```

Comment:
Just making clear that this error will be logged until the remote server is reachable again.

Reply:
Yes. I think that is fine, though a backoff loop may be appropriate if the host is down for longer than a few seconds.